.github/workflows/pull_request.yml

```diff
@@ -62,3 +63,10 @@ jobs:
 
       - name: Stop app
         run: make stop
+
+      - uses: actions/checkout@v2
```

Comment:
Do we need this all the time or was it just for debugging?

Reply:
It will only run when PR fails: https://github.com/CodeForAfrica/actNOW/blob/125c02bea200f4fb542c460e36a7a19012f91ba1/.github/workflows/pull_request.yml#L69